AmplitudeInteraction.py

```diff
@@ -13,40 +13,53 @@
 measures = {'uniques' : 'uniques', 'event totals' : 'totals', 'active %' : 'pct_dau', 'average' : 'average'}
 dfList = []
 keyFile = open(r'./.secret/api_keys.txt','r')
+keys = keyFile.read().split('\n')
+keyFile.close()
```

Comment:
Always use with open
That is much safer and guarantees closing files
Do that for all the files